indy_node/server/domain_req_handler.py

```diff
@@ -299,16 +309,14 @@ def _get_current_revoc_entry_and_revoc_def(self, author_did, revoc_reg_def_id, r
         return current_entry, revoc_def
 
     def _validate_revoc_reg_entry(self, req: Request):
+        author_did = req.identifier
         current_entry, revoc_def = self._get_current_revoc_entry_and_revoc_def(
-            author_did=req.identifier,
+            author_did=author_did,
             revoc_reg_def_id=req.operation[REVOC_REG_DEF_ID],
             req_id=req.reqId
         )
-
-        is_owner = False
-
-        if get_req_id(req) == req.operation[REVOC_REG_DEF_ID]:
-            is_owner = True
+        txn_data = get_payload_data(revoc_def)
+        is_owner = True if txn_data.get(CRED_DEF_ID) == author_did else False
```

Comment:
First note, it's better to write
` is_owner = txn_data.get(CRED_DEF_ID) == author_did`

Second note, a check for `txn_data.get(CRED_DEF_ID)` is incorrect here.
What you should do instead:
- get RevocDef'S ID (stet key) as `req.operation[REVOC_REG_DEF_ID]` 
- get RevocDef's author DID from RevocDef'S ID  (have a look at `make_state_path_for_revoc_def`, you need DID part from there). I'm not sure if we have a method to get this value from ID. You may consider creating one.
- compare the DID received ion the previous step to `author_did `